src/core/utils/fs.cpp

```diff
@@ -34,6 +34,8 @@
 #include <QFileInfo>
 #include <QSettings>
 #include <QCoreApplication>
+#include <QDesktopServices>
```

Comment:
Oh, right, that's how the original patch was made. I got rid of the ifdefs for some reason.